src/server_manager/web_app/gcp_server.ts

```diff
@@ -31,6 +33,39 @@ enum InstallState {
   DELETED
 }
 
+export function getRegionId(zoneId: ZoneId): string {
```

Comment:
Move this to the `Zone` class.

Reply:
This would be the geolocation property of the `CloudLocation`

Reply:
This is still needed, but it's only used by gcp_server and gcp_account.